aws/aws_vpc/simple/main.tf

```diff
@@ -23,5 +23,5 @@ provider "aws" {
 
 # Documentation: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/vpc
 resource "aws_vpc" "changeme_aws_vpc_simple" {
-  cidr_block = "10.0.0.0/16"
+  cidr_block = "10.1.0.0/16"
```

Comment:
Was this change to get past a left-over network?

Reply:
There were 2 examples using the same cidr: `"10.0.0.0/16"` so they both couldn't be applied at the same time.
But the bigger issue were some leftover resources (e.g. vpc) that I needed to cleanup manually.

I've also seen timeouts and jobs cancelations happening occasionally – that's probably where we got he leftover resources from. (Ie. job canceled before `terraform destroy` is done or due to timeout)